dev/devicelab/bin/tasks/flutter_gold_triage.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter_devicelab/framework/framework.dart';
+
+const String _kTriageCountKey = 'count';
+
+Future<TaskResult> getTriageCount() async {
+  final HttpClient skiaClient = HttpClient();
+  int digestCount = 0;
+  try {
+    await skiaClient.getUrl(Uri.parse(
+      'https://flutter-gold.skia.org/json/byblame?query=source_type%3Dflutter'
+    ))
+      .then((HttpClientRequest request) => request.close())
```

Comment:
Can you use await instead of the then closures?

Reply:
Done!